.github/workflows/pr-close-delete-env.yml

```diff
@@ -5,25 +5,61 @@ on:
     types: [closed]
   workflow_dispatch:
     inputs:
-      environment:
-        description: "Pull request number"
+      slotID:
+        description: "Pull request number (Add multiple IDs seperated by a space)"
```

Comment:
since the param name is singular - keep it singular

Reply:
Done

Reply:
param name is still singular